### DISCUSSION TITLE
Ask help for error installing

### DISCUSSION THREAD

**Discussion by Maveric33** [skyline-emu/skyline#245] [General]
Hello,
I am trying to install skyline for the first time on my HTC U11 Android 9.0 phone but I get the following error.

"an error occurred during package analysis"

Version: https://github.com/skyline-emu/skyline/actions/runs/1279164230

After several hours of research and testing here are the actions taken but without success

Installation of an unknown source
- Debug on USB
- No antivirus
- Attempt to extract complete error logs from the installation (Without success because my phone is not rooted)
- Attempt to install previous version
- Attempt to install app-debug then app-release
- Reboot of the phone
- Checking installation rights from my file explorer

I admit I'm starting to dry up a bit if someone would have an idea please?

Thanks in advance

**Comment by Maveric33** [comment]
I was able to install another installation from a different google play source, I did not encounter any problem.

**Comment by MythicalPlayz** [comment]
maybe it was corrupted download, however Skyline only supports Android 10+ so it may alsi be an outdated os

**Comment by Maveric33** [comment]
Thank you for your feedback,
It seems to me that I had seen 9.0+ in the info.

For the corruption I had to load it no less than 5 times so on that I think we are good.

If you are sure that it is now 10.0+ then it is normal this error.

Would you have the sources of the version indication by any chance please?

**Reply by PixelyIon** [reply]
It's Android 10+ now, we ended up increasing the minimum version to be more in line with the grade of phones which should be able to run commercial games properly.

**Reply by Maveric33** [reply]
Thx you very much for your feedback.

**Comment by Liankin616** [comment]
Im kind of new and dunno much my way around on how to download the emulator. Do older versions still works?  Like android 9 - 7? 
If older version still works can you guys give me the link please?
